mlir/lib/Target/LLVMIR/Dialect/ROCDL/ROCDLToLLVMIRTranslation.cpp

```diff
@@ -99,6 +99,7 @@ class ROCDLDialectLLVMIRTranslationInterface
       if (!llvmFunc->hasFnAttribute("amdgpu-flat-work-group-size")) {
         llvmFunc->addFnAttr("amdgpu-flat-work-group-size", "1,256");
       }
+      llvmFunc->addFnAttr("amdgpu-implicitarg-num-bytes", "256");
```

Comment:
Wasn't this in the other review as well?

Reply:
This PR depends on 79038 (the other review) and they both need to land together. When I created stacked PRs, it seems that Github brought both the commits together in this PR.

You may use [this ](https://github.com/llvm/llvm-project/pull/79039/commits/a66ac33975381b1acdae0c177842d2f711ad9ab9) to see changes of this review only.